nav2_bringup/bringup/params/nav2_multirobot_params_2.yaml

```diff
@@ -53,6 +53,10 @@ bt_navigator:
     global_frame: map
     robot_base_frame: base_link
     odom_topic: /odom
+    # groot ports need to be different from other robots config !!
+    enable_groot_monitoring: True
```

Comment:
Thanks for changing the ports. Can you default these to `False` instead for the robot 1 and robot 2? That way its not running in case someone actually needs to use it